README.md

```diff
@@ -1,12 +1,36 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The assignment is to create a business site containing hero header (image or video), a form with at least three different HTML input types. The site should be responsive and adapt well to mobile, tablet, and desktop.  
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+1. Hero header
+Just to find the hero image was a struggle. I found an image that I wanted to use - it was "gradient", went from purple to pink - but it wasn't for free. I asked ChatGPT if it could recreate the image for me, and it said that I could create it with CSS, which I wanted to try and it turned out great! But, it's not an image or video. So I commented it out, added the code to CodePen, and took a screenshot: https://codepen.io/joheri1/pen/vYqajdp. But changed my mind, and added it as an "overlay" (with the help from ChatGPT and Google) and added an image of a mountain as hero image (found here: https://stock.adobe.com/search?k=mountain+doodles). Used this site that could make the background transparant: https://www.remove.bg/. The gradient background still didn't work, and I spent a lot of time trying to figure out why, until I asked ChatGPT whom spotted that it was a tiny little space between linear-gradient and the parentheses that messed up the code. 
+
+2. The form questionnarie
+I decided to use a form on the internet as my inspiration (https://tjejerkodar.se/bli-medlem/). The form contains a html heading followed by an Unordered List, five oval input fields with the label text inside of each oval box, a check box ("I agree...") and an oval submit button at the end of the form. 
+
+The biggest issue with this was that I found it hard to understand/read the code in the inspector, so I had to google a lot. How to add the text inside of each input field ("placeholder"), how to add space around the boxes, etc. The form also contained things I couldn't understand, because they weren't visible to the eye, such as "<label for="frm_email_3">Om du är mänsklig, lämna det här fältet tomt.</label>". 
+
+3. Minor layout issues. 
+The name fields are not positioned the way I wanted, the radio button is oval and can't be unchecked, etc. Since time was running out, I started to write PostIts with all the flaws I wanted to correct, added some in a pile that need to be fixed before handing in my project (upload to Netlify, make the site responsive for different devices, etc), and some notes in a pile to fix later, if I have the time (like the oval radio button). 
```

Comment:
That's the behaviour of radio buttons, it's a group of buttons and you always have to pick exactly one, i.e.
Are you a:
- Lunch Learner
- Early Bird
- Evening Owl

We know that each user completing our form will be one - and one only - of these options. For checking/unchecking, it's better to use a checkbox. Checkboxes can be grouped as well, but for when the users can choose more than one option, i.e.
Are you interested in:
- Frontend development
- Backend development
- UX
- UI